src/pages/[platform]/prev/tools/libraries/index.mdx

```diff
@@ -1,37 +0,0 @@
-import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
```

Comment:
Ditto need to keep this content for reference for content update - we kept it out of the directory to hide it but still need it and agreed users could cross-link to it without breadcrumbs in meantime